<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefs_identity</class>
 <widget class="QWidget" name="prefs_identity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>375</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string>Fill in your identity and information about your translation team. This information is used when updating the header of a file.</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string comment="@label:chooser">Default language:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>DefaultLangCode</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string comment="@label:textbox">&amp;Email:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>kcfg_authorEmail</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel2_2">
     <property name="text">
      <string comment="@label:textbox">&amp;Name:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>kcfg_authorName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="kcfg_authorName">
     <property name="toolTip">
      <string>Your name, in English</string>
     </property>
     <property name="whatsThis">
      <string>Please enter here your name and surname written in English</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string comment="@label:textbox">&amp;Default mailing list:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_DefaultMailingList</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="kcfg_DefaultMailingList">
     <property name="toolTip">
      <string>The email of your team mailing list</string>
     </property>
     <property name="whatsThis">
      <string>Write the email of your translating team mailing list</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="KComboBox" name="DefaultLangCode">
     <property name="toolTip">
      <string>Language you translate to</string>
     </property>
     <property name="whatsThis">
      <string>Set the default language you are going to translate to</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="kcfg_authorEmail">
     <property name="toolTip">
      <string>Write your email</string>
     </property>
     <property name="whatsThis">
      <string>Write your email here so it will appear in the po file header with your name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="LangCodeSaver" name="kcfg_DefaultLangCode">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="kcfg_authorLocalizedName">
     <property name="toolTip">
      <string>Your name in your own language</string>
     </property>
     <property name="whatsThis">
      <string>Write your name and surname in your language with your language alphabet.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string comment="@label:textbox">Localized name:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_authorLocalizedName</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="kcfg_overrideLangTeam">
     <property name="text">
      <string>Default Language-Team:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="kcfg_userLangTeam">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>LangCodeSaver</class>
   <extends>QLineEdit</extends>
   <header>prefs.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_authorName</tabstop>
  <tabstop>kcfg_authorEmail</tabstop>
  <tabstop>DefaultLangCode</tabstop>
  <tabstop>kcfg_DefaultMailingList</tabstop>
  <tabstop>kcfg_authorLocalizedName</tabstop>
  <tabstop>kcfg_DefaultLangCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
